test/degreed/content/courses_test.rb

```diff
@@ -86,33 +91,35 @@ def test_get_courses_filtered_by_external_id
                   "self": "https://api.degreed.com/api/v2/content/courses",
                   "next": "https://api.degreed.com/api/v2/content/courses?next=OomeEL"
                 },
-                "data": {
-                  "type": "content/courses",
-                  "id": "foo",
-                  "attributes": {
-                    "provider-code": null,
-                    "external-id": "arstaroisen",
-                    "degreed-url": "https://degreed.com/courses/?d=V9RVZY40PJ",
-                    "title": "New Course",
-                    "summary": "A Summary",
-                    "url": "https://dev.lessonly.com",
-                    "obsolete": false,
-                    "image-url": null,
-                    "language": null,
-                    "duration": 200,
-                    "duration-type": "Seconds",
-                    "cost-units": 0.0,
-                    "cost-unit-type": null,
-                    "format": null,
-                    "difficulty": null,
-                    "video-url": null,
-                    "created-at": "0001-01-01T00:00:00",
-                    "modified-at": "0001-01-01T00:00:00"
-                  },
-                  "links": {
-                    "self": "https://api.degreed.com/api/v2/content/courses/foo"
+                "data": [
```

Comment:
Data should actually contain an Array in the response to "list courses" https://api.degreed.com/docs/#get-all-courses